server/middleware/addServicesToRequest.ts

```diff
@@ -0,0 +1,10 @@
+import { RequestHandler } from 'express'
+import { Services } from '../services'
+
+export default function addServicesToRequest(services: Services): RequestHandler {
```

Comment:
It was used and put in so that it's easier to access them in the controllers and functions that have the req parsed into them. But we don't need to use it.

Reply:
this has been removed